data/json/field_type.json

```diff
@@ -1637,5 +1637,16 @@
     "half_life": "1 seconds",
     "phase": "solid",
     "display_field": false
+  },
+  {
+    "id": "fd_churned_earth",
+    "type": "field_type",
+    "intensity_levels": [ { "name": "churned earth", "sym": "^", "color": "brown", "dangerous": false } ],
+    "half_life": "1 seconds",
```

Comment:
Would probably make sense to bump this up, maybe a few hours. If the point of the PR is to keep burrowing monster from defacing terrain for ever you could even bump it to a few days.

Reply:
I'll bump up the duration. is there a way to account for field lifetime seeming to get locked in place when the field is out of the reality bubble to prevent it from happening? I'd like to avoid that if at all possible

Reply:
Field not getting processed out of the bubble probably deserved an issue, I think it could/should be solved like fo fires. Waiting for that to get fixed you can still bump the half life a bit, it will disapear if players stay close and that's when it matters.

Reply:
got it - I'll push a commit once I get some time